Source/Core/VideoBackends/Vulkan/VKRenderer.cpp

```diff
@@ -315,9 +330,9 @@ void Renderer::BindBackbuffer(const ClearColor& clear_color)
     }
 
     res = m_swap_chain->AcquireNextImage();
+    if (res != VK_SUCCESS)
```

Comment:
Did you move this inside the `if` intentionally?

Reply:
Yep. The outer block already checks for a failed result, so it would be impossible for it to fail twice outside of it.